_doc-gen-robots/action.yml

```
@@ -72,6 +79,12 @@ runs:
             echo "Disallow: /version/${version}" >> robots.txt
         done
 
+    - name: "Disallow the pull request documentatio directory"
+      if: ${{ inputs.disallow-pr == 'true' }}
+      shell: bash
+      run: |
+        echo "Disallow: /pr/" >> robots.txt
```

Note that this is named `pull`:

```suggestion
        echo "Disallow: /pull/" >> robots.txt
```

---

Thanks for catching this!